tests/unit/plugin.spec.js

```diff
@@ -42,7 +102,7 @@ describe("CLD plugin", () => {
       const localVue = createLocalVue();
       localVue.use(Cloudinary, { components: [CldImage] });
 
-      const isImageComponentInstalled = Object.hasOwnProperty.call(localVue.options.components, 'CldImage');
+      const isImageComponentInstalled = Object.hasOwnProperty.call(localVue.options.components, 'cld-image');
```

Comment:
why is cld-image changed here, but cldVideo not?

Reply:
there is no need. but yes sure we can change it just to make sure no `cld-video` is presented.

Reply:
Might as well be consistent :)